src/components/sections/favorite-departure-item.tsx

```diff
@@ -88,9 +88,10 @@ function FavoriteItemContent({favorite, icon, ...props}: BaseProps) {
 
 const favoriteStyle = StyleSheet.create({
   flexStart: {
-    alignItems: 'flex-start',
+    alignItems: 'center',
+    justifyContent: 'center',
   },
   favorite__icon: {
-    minWidth: 30,
+    marginRight: 12,
```

Comment:
Burde dette hentes ut fra theme?

Reply:
Hehe, du var 10s raskere enn meg å påpeke det. Hva syns du?

Reply:
Tenker det er greit å bruke hook for å få det konsekvent med resten av systemet.

Reply:
Må gjøre det om til en hook, og så forandre på en del andre styles i komponenten.

Reply:
Den er god

Reply:
Hm. Hvorfor må du endre styles? Dersom du gjør om til å lage en hook og kaller verdien du lager `favoriteStyle` burde vel de referansene som blir brukt i den komponenten rett?

Reply:
Ser at det ikke ble så mye som jeg trodde, oppdatert nå.

Ser det ok ut?

Reply:
Ja, bare oppdatere denne da 😛  Kunne sendt inn suggestion men kommer ikke på rett property navn her. 

```
    marginRight: 12,
```

Reply:
🤦

Reply:
Nå sitter'n